src/v2/api/controllers/AuthController.ts

```diff
@@ -160,13 +176,30 @@ export class AuthController {
     return this.userService.getUser(req.user.id);
   }
 
+  @ApiOkResponse()
+  @ApiBadRequestResponse({
+    description: `\n
+    NotRegisteredException:
+      This email is not registered yet`,
+  })
+  @ApiTooManyRequestsResponse({
+    description: `\n
+    TooManyActionsException:
+      Too many actions. Try later`,
+  })
   @Post('/forgotPassword')
   async forgotPassword (
     @Body() body: ForgotPasswordDTO,
   ) {
     return this.authService.forgotPassword(body.email);
   }
 
+  @ApiOkResponse()
```

Comment:
describe the request body (ResetPasswordDTO)

Reply:
nothing is returned